webapp/src/editortoolbar.tsx

```diff
@@ -208,6 +209,39 @@ export class EditorToolbar extends data.Component<ISettingsProps, EditorToolbarS
         this.props.parent.pairAsync();
     }
 
+    protected onCannotPairClick = async () => {
+        pxt.tickEvent("editortools.pairunsupported", undefined, { interactiveConsent: true });
+        const reasonUnsupported = await pxt.usb.getReasonUnavailable();
+        let modalBody: string;
+        switch (reasonUnsupported) {
+            case "security":
+                modalBody = lf("WebUSB appears to have been disabled by browser policy; check with your technical adminstrator for details.");
```

Comment:
Can this language be simplified for clarity? It felt wordy to me.

```suggestion
                modalBody = lf("WebUSB is disabled. Check with your admin for help.");
```

Reply:
I do like simplifying it like this, but I'd think some mention of 'browser policy' somewhere is still probably useful -- web usb is not a particularly common feature so it's very possible it won't be top of mind for the admin if they're asked about it - e.g. maybe it was disabled a few years back due to the chrome cve (I think 2018ish?) and it's been forgotten about since then, or another person set it up. How about:

```suggestion
                modalBody = lf("WebUSB is disabled by browser policies. Check with your admin for help.");
```

maybe?

Reply:
makes sense.